roles/openshift_logging_eventrouter/defaults/main.yaml

```diff
@@ -0,0 +1,7 @@
+---
+openshift_logging_eventrouter_replicas: 1
+openshift_logging_eventrouter_sink: stdout
+openshift_logging_eventrouter_nodeselector: ""
```

Comment:
Should this default to empty hash?

Reply:
majority of roles also default nodeselector to an empty string, I was trying to be consistent with how other roles are implemented.

- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_curator/defaults/main.yml#L11
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_elasticsearch/defaults/main.yml#L8
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_kibana/defaults/main.yml#L10


Mux and fluentd don't, should I follow rather this pattern?
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_fluentd/defaults/main.yml#L10
- https://github.com/openshift/openshift-ansible/blob/d57f27bf71f68ba91996bc4a0b1e79b3be956b7e/roles/openshift_logging_kibana/defaults/main.yml#L10